policy/deprecated_v7.go

```diff
@@ -390,8 +392,10 @@ func (s *DeprecatedV7_ScoringSpec) ApplyToV8(ref *explorer.Mquery) {
 
 func (d *DeprecatedV7_PolicySpec) ToV8() *PolicyGroup {
 	policies := make([]*PolicyRef, len(d.Policies))
-	i := 0
```

Comment:
nit: I think, you should also remove the `i++` at the end of the three loops. Otherwise, it is confusing to read.

Reply:
great catch, I missed that while converting